simple_certmanager/models.py

```diff
@@ -60,6 +70,21 @@ class Meta:
             )
         ]
 
+    def create_certificate(self, certificate):
```

Comment:
```suggestion
    @transaction.atomic
    def create_certificate(self, certificate):
```
since you're doing multiple writes to the DB in a single method, best to wrap it in an atomic block so that all or nothing succeeds if there are errors